fv/fv_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build fvtests
+
+package fv_test
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+func ConditionIt(cond bool, title string, code func()) {
+	if cond {
+		It(title, code)
```

Comment:
Can see one downside of this now I see it written out; when a test fails, it'll report this line number instead of the "real one".  That said; I tend to look for the line number of the assert, which should still be correct.

Reply:
thank can be fixed :+1:  (iirc)

Reply:
I couldn't see a variant of `It` with an offset when I looked.  One option would be to do:
```
_ = testIfTCP && It(...)
```
since `It()` returns `true`. Bit of a hack though?

Reply:
I did not know that It() returns anything until I digged deeper in ginkgo, but it is not too bad. And it makes the lines shorter, so I guess it is a winner! :-)